packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: snowplow/snowplow_utils
-    version: [">=0.11.0", "<0.12.0"]
+    version: [">=0.12.0", "<0.13.0"]
```

Comment:
could this be `version: [">=0.11.0", "<0.13.0"]` instead? I figure the changes between 0.12 and 0.11 are non-breaking so both would work?
I suggest this as it sort of feels like the change as is should bump this to 0.6.0, whereas if we can just widen it (so both 0.11 and 0.12 work) then 0.5.4 is fine.

Reply:
makes sense 👍

Reply:
have amended commit with this change